src/rust/src/params.rs

```diff
@@ -62,6 +63,7 @@ pub(crate) fn network(
             canopy: i32_to_optional_height(canopy),
             nu5: i32_to_optional_height(nu5),
             nu6: i32_to_optional_height(nu6),
+            // TODO: NU6.1
```

Comment:
`zcashd` is currently based on an old version of the Zcash Rust crates. We will either need to get it up to latest releases, or maintain a `zcashd`-specific branch that we can patch in.